components/Chips/src/MDCChipView.m

```diff
@@ -609,6 +638,13 @@ - (void)layoutSubviews {
         [UIBezierPath bezierPathWithRoundedRect:self.bounds cornerRadius:cornerRadius].CGPath;
   }
 
+  CAShapeLayer *shapedLayer = (CAShapeLayer *)_chipView.layer.mask;
+  if (![shapedLayer isKindOfClass:[CAShapeLayer class]]) {
+    shapedLayer = [CAShapeLayer layer];
+    _chipView.layer.mask = shapedLayer;
+  }
+  shapedLayer.path = self.layer.shadowPath;
```

Comment:
Was this intentionally added? It doesn't look relevant to the change.

Reply:
oh yeah, this is needed to have the _chipview shape be the same as the chip. If you think this is not clear I could add a ocmment.